css/footer.css.map

```diff
@@ -0,0 +1,9 @@
+{
+    "version": 3,
+    "mappings": "AAAA,AAAA,OAAO,CAAC;EACJ,OAAO,EAAE,WAAW;EACpB,OAAO,EAAE,IAAI;EACb,cAAc,EAAE,MAAM;EACtB,WAAW,EAAE,MAAM;EACnB,UAAU,EAAE,MAAM;CAmBrB;;AAlBI,AAAD,YAAM,CAAC;EACH,OAAO,EAAE,IAAI;EACb,eAAe,EAAE,YAAY;EAC7B,WAAW,EAAE,MAAM;CACtB;;AACA,AAAD,gBAAU,CAAC,CAAC,CAAC;EACT,eAAe,EAAE,aAAa;EAC9B,cAAc,EAAE,GAAG;EACnB,WAAW,EAAE,MAAM;EACnB,OAAO,EAAE,OAAO;EAChB,WAAW,EAAE,IAAI;EACjB,YAAY,EAAE,IAAI;CACrB;;AAlBL,AAoBI,OApBG,CAoBH,GAAG,CAAC;EACA,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,KAAK;CAChB",
+    "sources": [
+        "../src/styles/footer.scss"
+    ],
+    "names": [],
+    "file": "footer.css"
```

Comment:
do you know why both the .css and the .scss are needed? just having trouble figuring out their difference...

Reply:
Nah that was just an auto create from my live sass compiler that I didnt delete 😄